lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
```

Comment:
@Fishrock123 @kzc pulled the logic into its own module, gives us a nice central place to address any other edge-cases that come up, without the code hanging out in a bunch of random libraries:

https://github.com/yargs/set-blocking